kentik_api_library/setup.py

```diff
@@ -239,6 +239,34 @@ def run(self):
         print(*packages, sep="\n")
 
 
+class SetupAPIClientVersion(Command):
```

Comment:
This is not needed. [setuptools_scm](https://github.com/pypa/setuptools_scm) can provide version.
You just need to add:
```
from setuptools_scm import get_version
```
and call `get_version()` where necessary.

Reply:
I've also found out that `setuptools_scm` older than 7.0 has a bug that prevents it from working in this project (and I'd say any other too :-(). We will need to add:
```
[build-system]
requires = ["setuptools_scm>=7.0", "wheel"]
```
to `pyproject.toml` ... and after that we can also remove the `relative_to` directive from the `[tool.setuptools_scm]` section (as you've proposed some time ago).

Reply:
Updating

Reply:
You are still working on it, or  do you consider it done?

Reply:
Done. Or do you have further suggestions?

Reply:
Well, you've really not addressed my comments. I'll merge the code and address them myself.